cfn-resources/project/Makefile

```diff
@@ -11,12 +11,12 @@ LINKER_FLAGS=-s -w -X github.com/mongodb/mongodbatlas-cloudformation-resources/u
 build:
 	@echo "==> Building handler binary"
 	cfn generate
-	env GOOS=$(goos) CGO_ENABLED=$(cgo) GOARCH=$(goarch) go build -ldflags="$(LINKER_FLAGS)" -tags="$(tags)" -o bin/handler cmd/main.go
+	env GOOS=$(goos) CGO_ENABLED=$(cgo) GOARCH=$(goarch) go build -ldflags="$(LINKER_FLAGS)" -tags="$(tags)" -o bin/bootstrap cmd/main.go
 
 debug:
 	@echo "==> Building handler binary for debugging"
 	cfn generate
-	env GOOS=$(goos) CGO_ENABLED=$(cgo) GOARCH=$(goarch) go build -gcflags="$(DEBUG_FLAGS)" -ldflags="$(LINKER_FLAGS)" -tags="$(tags)" -o bin/handler cmd/main.go
+	env GOOS=$(goos) CGO_ENABLED=$(cgo) GOARCH=$(goarch) go build -gcflags="$(DEBUG_FLAGS)" -ldflags="$(LINKER_FLAGS)" -tags="$(tags)" -o bin/bootstrap cmd/main.go
```

Comment:
updating to work with latest versions of cloudformation-cli-go-plugin as per https://github.com/aws-cloudformation/cloudformation-cli/issues/1053#issuecomment-1904375299 
Currently this is resulting in errors when trying to submit resource to private registry and manually creating a stack [`Lambda function handler threw an uncaught exception: fork/exec /var/task/handler: no such file or directory`]